Documentation/ApiOverview/Internationalization/Introduction.rst

```diff
@@ -56,3 +60,20 @@ whose :php:`translate()` method also takes an array as argument and runs PHP's
 :php:`vsprintf()` on the localized string. However, in the future it is expected this Extbase
 specific class will melt down and somehow merged into the core API classes to get rid of this
 duplication.
+
+Label Access in Fluid
+=====================
+
+In PHP, a typical call to fetch a string in the language selected by a user looks like this:
+
+.. code-block:: html
+
+   <f:translate key="key1" extensionName="SomeExtensionName" />
+   // or inline notation
+   {f:translate(key: 'someKey', extensionName: 'SomeExtensionName')}
+
+If the correct context is set, the current extension name and language is provided by the request. Otherwise it must be provided.
+
+TODO
+The documentation for the ViewHelper can be found at https://docs.typo3.org/other/typo3/view-helper-reference/9.5/en-us/typo3/fluid/latest/Translate.html
```

Comment:
how can this be linked correctly?

Reply:
```
:ref:`t3viewhelper:typo3-fluid-translate`.

```

Make sure shortcut t3viewhelper is in Settings.cfg. (I checked and it is not)

Settings.cfg (for 9.5):

`t3viewhelper = https://docs.typo3.org/other/typo3/view-helper-reference/9.5/en-us/`

Settings.cfg (for master):

`t3viewhelper = https://docs.typo3.org/other/typo3/view-helper-reference/master/en-us/`